### PR TITLE
Update repository URL after repository migration

### PR DESCRIPTION
**Pull request by wangxin**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
The sonic-mgmt repository has been migrated to new location:
    https://github.com/sonic-net/sonic-mgmt

The URLs embedded in docs and code need to be updated for this change.

#### How did you do it?
This change updated all the URLs in docs and code.

This change also merged a duplicated entry in test_mark_conditions.yaml:
* cacl/test_cacl_application.py::test_multiasic_cacl_application

### CODE REVIEW

File: ansible/roles/test/files/ptftests/py3/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Migration includes 4 stages:
 ## Stage 1: Prepare Python3 virtual environment in docker-ptf
 Add Python3 virtual environment in docker-ptf, we will keep Python2 in this stage for incremental migration.
 
-[PR](https://github.com/Azure/sonic-buildimage/pull/10599) to address this.
+[PR](https://github.com/sonic-net-net/sonic-buildimage/pull/10599) to address this.
```
Comment (yejianquan) [inline ansible/roles/test/files/ptftests/py3/README.md:21]: Shoud be sonic-net

File: ansible/roles/test/files/ptftests/fib_test.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 '''
 Description:    This file contains the FIB test for SONIC
 
-                Design is available in https://github.com/Azure/SONiC/wiki/FIB-Scale-Test-Plan
+                Design is available in https://github.com/sonic-net-net-net/SONiC/wiki/FIB-Scale-Test-Plan
```
Comment (yejianquan) [inline ansible/roles/test/files/ptftests/fib_test.py:4]: Shoud be sonic-net

File: ansible/roles/test/files/acstests/acltb_test.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 Description:
 
     This file contains the ACL test for SONiC testbed
-    Implemented according to the https://github.com/Azure/SONiC/wiki/ACL-test-plan
+    Implemented according to the https://github.com/sonic-net-net/SONiC/wiki/ACL-test-plan
```
Comment (yejianquan) [inline ansible/roles/test/files/acstests/acltb_test.py:5]: Shoud be sonic-net

File: ansible/roles/test/files/ptftests/py3/README.md (hunk 1 above)
Comment (yxieca) [inline ansible/roles/test/files/ptftests/py3/README.md:21]: Another one here.

**Comment by wangxin** [comment]
Thanks for pointing out these errors. I counted on my editor to do the search and replace. It looks like a weird bug of my editor...

**Review by yejianquan — APPROVED** [review]
LGTM

**Pull request merged**